query-engine/prisma-models/src/projections/model_projection.rs

```diff
@@ -15,6 +14,27 @@ impl From<Field> for ModelProjection {
     }
 }
 
+impl From<FieldSelection> for ModelProjection {
+    fn from(fs: FieldSelection) -> Self {
+        (&fs).into()
+    }
+}
+
+/// [Composites] todo: Temporary converter.
+impl From<&FieldSelection> for ModelProjection {
+    fn from(fs: &FieldSelection) -> Self {
+        Self {
+            fields: fs
+                .selections()
```

Comment:
It might be better to make this a function on the `FieldSelection` struct. something like`get_scalar_fields()`

Reply:
there already is: `as_scalars`. We only do this explicitly here because we don't want to error if there's something besides scalars in the selection, as this is only a temporary converter that will be deleted together with `ModelProjection` down the line.